src/makeRelativeDate.README.md

```diff
@@ -0,0 +1,23 @@
+
+# `makeRelativeDate`
```

Comment:
rails has similar helper called [`time_ago_in_words`](https://api.rubyonrails.org/v5.2/classes/ActionView/Helpers/DateHelper.html#method-i-time_ago_in_words), which i think is a great description of what this does. 

what do you think about using that here?

reason i ask is i think the current name is ambiguous:

- what is the relative date it offers me? it could be as simple as `Date.now()` minus the time you pass in
- what does 'make' do here? is it going to return something or produce a side effect?

it's not really 'making' a date, it generates and returns a english sentence describing a duration before now

`timeAgoInWords(time, opts)`, seems to indicate that well, to me?

Reply:
I spent some time thinking about this but ending up just going with what it was called before. I didn't think of this name though, that's a really nice api!